### DISCUSSION TITLE
plotting not working

### DISCUSSION THREAD

**Discussion by Tanukiman11** [Chia-Network/chia-blockchain#1151] [Q&A]
I upgraded to the new node v. 0.2.4 now when I try to plot and view the log it gives an error (see screenshot)
![Capture](https://user-images.githubusercontent.com/80014086/109995324-6df65e80-7ccb-11eb-9e3a-43fba94c7be6.PNG)
... Not sure what to do?

Thank you in advance.

**Comment by joe11111-sudo** [comment]
![1619723927139](https://user-images.githubusercontent.com/74581472/116606106-d035ad80-a962-11eb-937a-6f75408c2358.jpg)
Why finished plotting, but came out not available
